Assets/Character/Scripts/Constants.cs

```diff
@@ -4,4 +4,10 @@
 	public const int BASE_JUMP_SPEED = 300;
     public const int BASE_ROLL_SPEED = 100;
     public const float ROLL_TIME = 1f;
+
+
+    public const int CONSUMABLE_COUNT = 3;

```

Comment:
Each consumable type (I don't know how many we'll have, but let's use health and stamina potions as an example) will have it's own duration. So the  constants should be for each type, and not the bass class. Example: stamina potions will might have a lower cooldown than health potions.